mmv1/products/apphub/Workload.yaml

```diff
@@ -0,0 +1,241 @@
+            name: type
+            description: 'Criticality type. '
+            required: true
+            values:
+              - :MISSION_CRITICAL
+              - :HIGH
+              - :MEDIUM
+              - :LOW
+        name: criticality
+        description: 'Criticality of the Application, Service, or Workload '
+      - !ruby/object:Api::Type::NestedObject
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: type
+            description: 'Environment type. '
+            required: true
+            values:
+              - :PRODUCTION
+              - :STAGING
+              - :TEST
+              - :DEVELOPMENT
+        name: environment
+        description: 'Environment of the Application, Service, or Workload '
+      - !ruby/object:Api::Type::Array
+        name: developerOwners
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: displayName
+              description: 'Optional. Contact''s name. '
+            - !ruby/object:Api::Type::String
+              name: email
+              description: 'Required. Email address of the contacts. '
+              required: true
+        description: 'Optional. Developer team that owns development and coding. '
+      - !ruby/object:Api::Type::Array
+        name: operatorOwners
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: displayName
+              description: 'Optional. Contact''s name. '
+            - !ruby/object:Api::Type::String
+              name: email
+              description: 'Required. Email address of the contacts. '
+              required: true
+        description: 'Optional. Operator team that ensures runtime and operations. '
+      - !ruby/object:Api::Type::Array
+        name: businessOwners
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: displayName
+              description: 'Optional. Contact''s name. '
+            - !ruby/object:Api::Type::String
+              name: email
+              description: 'Required. Email address of the contacts. '
+              required: true
+        description: 'Optional. Business team that ensures user needs are met and value
+          is delivered '
+    name: attributes
+    description: 'Consumer provided attributes. '
+  - !ruby/object:Api::Type::String
+    name: createTime
+    description: 'Output only. Create time. '
+    output: true
+  - !ruby/object:Api::Type::String
+    name: updateTime
+    description: 'Output only. Update time. '
+    output: true
+  - !ruby/object:Api::Type::String
+    name: uid
+    description: "Output only. A universally unique identifier (UUID) for the `Workload`
+      in the UUID4\nformat. "
+    output: true
+  - !ruby/object:Api::Type::Enum
```

Comment:
For output-only enums, we usually define them as strings, so that we don't need to keep a client-side list of possible values.